packages/next-app/.env.example

```diff
@@ -2,4 +2,5 @@ NEXTAUTH_URL=http://localhost:3000
 NEXT_PUBLIC_NEXTAUTH_SECRET=
 NEXT_PUBLIC_TWITTER_ID=
 NEXT_PUBLIC_TWITTER_SECRET=
-REDIS_URL=
\ No newline at end of file
+NEXT_PUBLIC_DISCORD_ID=
+NEXT_PUBLIC_DISCORD_SECRET=
```

Comment:
Avoid using `NEXT_PUBLIC_` suffix for any SECRET stuffs as it will be exposed to the client side